### PR TITLE
Update chromium style to match what's used in Chromium

### PR DESCRIPTION
**Pull request by agrieve**
Adds a "YAPF" style in order to avoid reformatting the YAPF codebase
and updating test expectations

Relevant Chromium style discussion:
https://groups.google.com/a/chromium.org/g/chromium-dev/c/RcJgJdkNIdg/discussion

### CODE REVIEW

**Comment by agrieve** [comment]
Following chromium code review comments (switched fully to PEP8 since 79 vs 80 characters was the only difference), I've change the CL to just remove "chromium" as a separate style.

Added CHANGELOG entry.

File: README.rst [hunk 1]
```diff
@@ -178,8 +178,8 @@ with a ``[yapf]`` heading. For example:
 
 The ``based_on_style`` setting determines which of the predefined styles this
 custom style is based on (think of it like subclassing). Four
-styles are predefined: ``pep8`` (default), ``chromium``, ``google`` and 
-``facebook`` (see ``_STYLE_NAME_TO_FACTORY`` in style.py_).
+styles are predefined: ``pep8`` (default), ``google`` and ``facebook`` (see
```
Comment (gwelymernans) [inline README.rst]: Please add ``yapf`` to the list of predefined styles.

**Comment by atetubou** [comment]
I think it is not good to replace style name atomically.
This makes it bit hard to migrate existing .style.yapf's for chromium.
https://cs.chromium.org/search/?q=c:chromium+f:style.yapf&sq=package:chromium&type=cs

**Comment by agrieve** [comment]
I'm curious what you think would work better? A warning? If we silently support "chromium" as a style, no one will migrate.

**Comment by atetubou** [comment]
I think it is better to have at least one release which has both yapf and chromium style in
https://github.com/google/yapf/releases
So that we can migrate our style files while using such release.

**Comment by agrieve** [comment]
I think style=yapf is a bit of an obscure thing to transition to. I was hoping projects would transition to style=pep8, which has been available for a long time.

If you want to not have the style change at all, I think it's probably best to transition to what style=yapf sets explicitly, which I believe is:

    based_on_style = google
    ALLOW_MULTILINE_DICTIONARY_KEYS = True
    ALLOW_SPLIT_BEFORE_DEFAULT_OR_NAMED_ASSIGN = False
    INDENT_DICTIONARY_VALUE = True
    INDENT_WIDTH = 2
    SPLIT_BEFORE_BITWISE_OPERATOR = True
    SPLIT_BEFORE_DOT = True
    SPLIT_BEFORE_EXPRESSION_AFTER_OPENING_PAREN = True

**Comment by gwelymernans** [comment]
@atetubou is correct though. I shouldn't have removed a style without a deprecation and migration plan and adjustment period. I'm sorry about that. It's not a good user experience...

**Pull request merged**